### DISCUSSION TITLE
How to enable CompiledBindings default for all the axaml files

### DISCUSSION THREAD

**Discussion by laolarou726** [AvaloniaUI/Avalonia#10261] [Q&A]
rt

**Comment by maxkatz6** [comment]
Starting with Avalonia "11.0-preview5" it's possible with:
```xml
<PropertyGroup>
   <AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>
</PropertyGroup>
```
in your csproj file.

Note, Rider doesn't understand this property yet, so it won't highlight syntax. But bindings still will be validated in compile time as expected.

**Reply by laolarou726** [reply]
When I trying to cast in XAML. Compiler throws an exception:
<img width="1341" alt="image" src="https://user-images.githubusercontent.com/25716486/217455237-f7c13787-74da-4fd3-b567-e14cbbc938fc.png">

<img width="1091" alt="image" src="https://user-images.githubusercontent.com/25716486/217455260-3bcec925-90a6-4003-941d-54d4a4e298be.png">

**Reply by laolarou726** [reply]
Is this a bug or I'm using wrong?

**Reply by maxkatz6** [reply]
It's a c#-like casting syntax. I.e. `Path=((asd:AddFriendViewModel)DataContext).IsAdding`

**Reply by laolarou726** [reply]
I see. Thanks a lot!
